packages/firebase_ml_vision/test/firebase_ml_vision_test.dart

```diff
@@ -141,12 +140,12 @@ void main() {
 
         final Barcode barcode = barcodes[0];
         expect(barcode.valueType, BarcodeValueType.unknown);
-        expect(barcode.boundingBox, const Rectangle<int>(1, 2, 3, 4));
+        expect(barcode.boundingBox, Rect.fromLTWH(1.0, 2.0, 3.0, 4.0));
```

Comment:
With implicit int -> conversion in Dart 2.1 could change to ints, unless backwards compatibility a requirement

Reply:
I think keeping backwards compatibility would be the best move.

The [current](https://github.com/flutter/plugins/blob/master/packages/firebase_ml_vision/pubspec.yaml#L24) version only requires `2.0.0` and upgrading to `2.1.0` might not be worth causing another breaking change. The only gain from using `2.1` would make the tests require slightly less code.